cmd/coreinstance/create_core_instance_k8s.go

```diff
@@ -166,3 +182,14 @@ func newCmdCreateCoreInstanceOnK8s(config *cfg.Config, testClientSet kubernetes.
 
 	return cmd
 }
+
+// GetK8sYaml strips empty properties which would typically be marshalled with yaml.Marshal
+func GetK8sYaml(req interface{}) {
+	out, _ := json.Marshal(req)
+	input := strings.NewReader(string(out))
+	var output strings.Builder
+	if err := json2yaml.Convert(&output, input); err != nil {
+		log.Fatalln(err)
```

Comment:
I'm not fan of panic... Maybe you can log.Print and return instead. Panic will give you a stack trace.

Reply:
done